README.md

```diff
@@ -7,6 +7,47 @@
 This repository contains Giant Swarm alerting and recording rules
 
 
+### Testing
+
+By creating unit tests for Alerting rules it's possible to get early feedback about possible misbehavior in alerting rules.
+Unit tests are executed via `promtool` (part of `prometheus`).
+
+By running `make test` in your local environment, all required binaries will be downloaded and tests will be executed.
+
+#### Writing new tests
+
+1. remove the rules file you would like to test from `test/conf/promtool_ignore`
```

Comment:
Should be the other way around IMO.
The current behaviour for each rule is to:
1- check - does not need any test file
2- test - skipped if there is no test file

So why not check all rule files by default?
In my tests, I only need an ignore entry for rules that helm complains about with:
```
Error: could not find template templates/alerting-rules/aws.management-cluster.rules.yml in chart
```

BTW, this makes me wonder if the list of ignored rules may be different for each provider ?